### DISCUSSION TITLE
How to get uploaded file object for editing ?

### DISCUSSION THREAD

**Discussion by hazwan91** [livewire/livewire#5064] [1. Help]
Hello.

I want to repopulate the images that have been successfully stored before for editing purpose. I can get the images of course. However i can't find a way to get the object like this like i get for uploading.

![Capture4](https://user-images.githubusercontent.com/97268905/175492553-175766c7-2968-461c-84c6-f68f19825c36.PNG)

I only receive a string to display the image from the database. I can however do a script to display the image but i have to change most of my code like fo example `$ic_image->extension()` or `$ic_image->getClientOriginalName` won't work anymore since i don't have that object similar like the one i use for an upload. I can change but if the user reupload new images again, then i have to use `$ic_image->extension()` again. Know what i mean ?

Is there any better practice or better way to do for editing our data that includes an uploaded file object ?

**Comment by joshhanley** [comment]
@hazwan91 that object will only exist while the image has been temporarily handled by Livewire. Once you persist it, you need to handle it yourself. 

So yeah it does mean having some conditionals in your code if you are displaying stored images and temporary uploads in the same place.

Hope this helps!
